guide/examining-obs-seq-final.rst

```diff
@@ -0,0 +1,24 @@
+Examining the obs_seq.final file
+================================
+
+If there are no changes in the model state after assimilation, then examine the
+``obs_seq.final`` file. There are two ways to do this.
```

Comment:
This should be moved to AFTER the two ways. We still want people to explore the observation sequence files even if there are no changes to the model state after assimilation.

I'd also like to vote to put in more content here - in addition to running obs_diag, they can explore with obs_seq_to_netcdf and various Matlab routines ... like link_obs.m

Reply:
I created a flow chart for the CAM6 Reanalysis paper, with many of the obs space diagnostics possibilities.
I've attached the CAM6 version, but could modify it for this doc.
[diagnostics_flowchart2.pdf](https://github.com/NCAR/DART/files/6313757/diagnostics_flowchart2.pdf)

Reply:
moved and explanded. Have not addressed or incorporated Kevins offer.